tests/translator/test_translator.py

```diff
@@ -349,14 +350,26 @@ def test_transform_success_openapi3(self, testcase, partition_with_region):
             output_fragment = transform(
                 manifest, parameter_values, mock_policy_loader)
 
-        print(json.dumps(output_fragment, indent=2))
```

Comment:
Thanks for adding the diff output, but this print is convenient when developing new features. As long as the output template is correct, we can copy/paste it as the output file for our unit tests.

Reply:
I tried adding back just the print, but we have this print logic in multiple functions in this test file, and I don't have time to do the refactor to include diff output everywhere. Since this change is to `test_transform_success_openapi3()`, which isn't even relevant to this PR (policy templates are tested by `test_transform_success()`), I went ahead and reverted the change to `test_templates.py` so we can get these policy template unit tests merged.